incognia_test.go

```diff
@@ -579,6 +579,7 @@ func (suite *IncogniaTestSuite) TestSuccessRegisterSignupWithParams() {
 		Address:        addressFixture,
 		AccountID:      postSignupRequestBodyWithAllParamsFixture.AccountID,
 		PolicyID:       postSignupRequestBodyWithAllParamsFixture.PolicyID,
+		ExternalID:     postSignupRequestBodyWithAllParamsFixture.ExternalID,
```

Comment:
This object was supposed to have all parameters defined. You can add `ExternalID: "external-id"` on line 103